docs/sources/reference/components/database_observability/database_observability.mysql.md

```diff
@@ -0,0 +1,61 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/database_observability.mysql/
+description: Learn about database_observability.mysql
+title: database_observability.mysql
+---
+
+Database Observability component. This component is under active development and can be run with Alloy flag `--stability.level=experimental`.
```

Comment:
```suggestion
<span class="badge docs-labels__stage docs-labels__item">Experimental</span>

# database_observability.mysql

{{< docs/shared lookup="stability/experimental.md" source="alloy" version="<ALLOY_VERSION>" >}}
```

Am I correct in assuming content for blocks/attributes will be added at some point?